scripts/generate_embeddings_openstack.py

```diff
@@ -53,17 +53,25 @@ def __init__(self, docs_path, base_url=ROOT_URL, version="18.0"):
         self.version = version
 
     def url_function(self, file_path: str):
+        # Document name mappings for cases where internal document names
+        # differ from published URL paths in Red Hat documentation
+        doc_mappings = {
```

Comment:
Is there no way to automatically get this mapping instead of maintaining it manually?